Romo/src/Drive/Telepresence/RMOpenTokManager.m

```diff
@@ -9,11 +9,12 @@
 #import "RMOpenTokManager.h"
```

Comment:
Since there is no OpenTok anymore let's rename or remove the corresponding classes.